pysearch/tests.py

```diff
@@ -0,0 +1,230 @@
+    {'keyword': 'soccer', 'weight': 5, 'url': 'url1', 'count': 100},
+    {'keyword': 'football', 'weight': 10, 'url': 'url2', 'count': 50},
+    {'keyword': 'soccer', 'weight': 5, 'url': 'url2', 'count': 25},
+    {'keyword': 'football', 'weight': 10, 'url': 'url3', 'count': 5},
+    {'keyword': 'soccer', 'weight': 5, 'url': 'url3', 'count': 5}
+]
+
+
+TEST_FILE = '<!doctype html>\n<html>\n<head>\n    <title>Example Domain</title>\n\n    <meta charset="utf-8" />\n    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />\n    <meta name="viewport" content="width=device-width, initial-scale=1" />\n    <style type="text/css">\n    body {\n<h2>this is h2</h2>        background-color: #f0f0f2;\n        margin: 0;\n        padding: 0;\n        font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;\n        \n    }\n    div {\n        width: 600px;\n        margin: 5em auto;\n  <h3>h3</h3>      padding: 50px;\n        background-color: #fff;\n        border-radius: 1em;\n    }\n    a:link, a:visited {\n        color: #38488f;\n <h2>this is h2</h2>       text-decoration: none;\n    }\n    @media (max-width: 700px) {\n        body {\n            background-color: #fff;\n        }\n        div {\n            width: auto;\n            margin: 0 auto;\n            border-radius: 0;\n            padding: 1em;\n        }\n    }\n    </style>    \n</head>\n\n<body>\n<div>\n    <h1>Example Domain</h1>\n<h3>h3</h3><h4>h4</h4><h5>h5</h5><h6>h6</h6>    <p>This domain is established to be used for illustrative examples in documents. You may use this\n    domain in examples without prior coordination or asking for permission.</p>\n    <p><a href="http://www.iana.org/domains/example">More information...</a></p>\n</div>\n</body>\n</html>\n'
+
+TEST_FILE_2 = '<!doctype html><html><head><title>Free Example Domain</title><meta charset="utf-8" /><meta http-equiv="Content-type" content="text/html; charset=utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" /><style type="text/css">body {background-color: #f0f0f2;margin: 0;padding: 0;font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;}h1 {color: red}h2 {color: red;}h3 {color: red;}h4 {color: red;}h5 {color: red;}h6 {color: red;}p {color: red;}div {width: 600px;margin: 5em auto;padding: 50px;background-color: #fff;border-radius: 1em;}a:link, a:visited {color: #38488f;text-decoration: none;}@media (max-width: 700px) {body {background-color: #fff;}div {width: auto;margin: 0 auto;border-radius: 0;padding: 1em;}}</style>    </head><body><div><h1>Example Domain</h1><p> the the the the the the the the marc marc marc marc marc marc pie pie pie pie pie pie pie glad glad glad glad free free free free free free only appear once </p><p><a href="http://www.iana.org/domains/example">More information...</a></p><h1>Header 1</h1><h2>marc Header 2</h2><h3>Header 3</h3><h4>Header 4</h4><h5>Header 5</h5><h6>Header 6</h6></div></body></html>'
+
```

Comment:
Instead of having raw HTML in your `tests.py`, you could always have a file with this HTML and read it in with `os.open` or something like it.